org.sofproject.gst.topo/src/org/sofproject/gst/topo/ui/wizards/GstNodeNewPage.java

```diff
@@ -78,7 +79,7 @@ public void createControl(Composite parent) {
         data.grabExcessHorizontalSpace = true;
         gstInspectToolCmd = new Text(inspectGroup, SWT.BORDER);
         gstInspectToolCmd.setLayoutData(data);
-        gstInspectToolCmd.setText(gstNode.getGstInspectToolCmd());
+        gstInspectToolCmd.setText(Platform.getPreferencesService().getString("org.sofproject.gst.topo.ui.preferences", "gstInspect", gstNode.getGstInspectToolCmd(), null));
         gstInspectToolCmd.addListener(SWT.Modify, confModifyListener);
 
 		Composite launchGroup = new Composite(control, SWT.NONE);
```

Comment:
I know that it's already here, but this indent is off. Can you make commit first that fixes this and then apply your changes on it?